python/tvm/topi/nn/depthwise_conv2d.py

```diff
@@ -38,20 +38,26 @@
         "out_filter",
         "hkernel",
         "wkernel",
-        "hpad",
-        "wpad",
+        "padt",
+        "padl",
+        "padb",
+        "padr",
+        "hdilation",
+        "wdilation",
         "hstride",
         "wstride",
     ],
 )
 
 
-def _get_workload(data, kernel, stride, padding, out_dtype):
+def _get_workload(data, kernel, stride, padding, dilation, out_dtype):
     """ Get the workload structure. """
     _, in_channel, height, width = [x.value for x in data.shape]
     channel, channel_multiplier, kh, kw = [x.value for x in kernel.shape]
     out_channel = channel * channel_multiplier
-    HPAD, WPAD, _, _ = get_pad_tuple(padding, kernel)
+    pt, pl, pb, pr = get_pad_tuple(padding, kernel)
+    hdilation, wdilation = dilation if isinstance(dilation, (tuple, list)) else (dilation, dilation)
```

Comment:
name them `dilation_h` and `dilation_w` as other places' code style

Reply:
Should this also apply to their names in the Workload tuples?  Or just the variables in the `_get_workload()` functions?

I named them `hdilation`, and `wdilation` in the Workload to follow the `"hstride"`, `"wstride",` style.

Reply:
Should be in the workload tuple too.

In fact, it should also be `stride_h` / `stride_w` in the code style.

Reply:
Thanks for the clarification, happy to change that too in my next commit.  While I'm at it, should I change `"hkernel"` and `"wkernel",` in the workloads to `kernel_h`, `kernel_w`?

Reply:
yeah.